jenkins/buildone.cmd

```diff
@@ -7,7 +7,9 @@
 setlocal
 
 REM set TEMP and TMP to a new temp folder under the WORKSPACE and create it
-set TEMP=%WORKSPACE%\TEMP
+pushd %WORKSPACE%\..\
+set TEMP=%CD%\TEMP

```

Comment:
I wanted the directory to be evaluated (rather than include ..) to avoid any hidden problems.